tiproxy/tiproxy-traffic-replay.md

```diff
@@ -89,16 +91,50 @@ Traffic replay is not suitable for the following scenarios:
     - `sample_replay_time`: the time when the SQL statement failed during replay. You can use this to view error information in the TiDB log file.
     - `count`: the number of times the SQL statement failed.
 
+    The following is example output from the `fail` table:
+
+    ```sql
+    SELECT * FROM tiproxy_traffic_replay.fail LIMIT 1\G
+    ```
+
+    ```
+    *************************** 1. row ***************************
+               cmd_type: StmtExecute
+                 digest: 89c5c505772b8b7e8d5d1eb49f4d47ed914daa2663ed24a85f762daa3cdff43c
+            sample_stmt: INSERT INTO new_order (no_o_id, no_d_id, no_w_id) VALUES (?, ?, ?) params=[3077 6 1]
+         sample_err_msg: ERROR 1062 (23000): Duplicate entry '1-6-3077' for key 'new_order.PRIMARY'
+         sample_conn_id: 1356
+    sample_capture_time: 2024-10-17 12:59:15
+     sample_replay_time: 2024-10-17 13:05:05
+                  count: 4
+    ```
+
     The `other_errors` table stores unexpected errors, such as network errors or database connection errors, with the following fields:
 
     - `err_type`: the type of error, presented as a brief error message. For example, `i/o timeout`.
     - `sample_err_msg`: the complete error message when the error first occurred.
     - `sample_replay_time`: the time when the error occurred during replay. You can use this to view error information in the TiDB log file.
     - `count`: the number of occurrences for this error.
 
+    The following is example output from the `other_errors` table:
+
+    ```sql
+    SELECT * FROM tiproxy_traffic_replay.other_errors LIMIT 1\G
+    ```
+
+    ```
+    *************************** 1. row ***************************
+              err_type: failed to read the connection: EOF
+        sample_err_msg: this is an error from the backend connection: failed to read the connection: EOF
+    sample_replay_time: 2024-10-17 12:57:39
+                 count: 1
+    ```
+
     > **Note:**
     >
-    > The table schema of `tiproxy_traffic_report` might change in future versions. It is not recommended to directly read data from `tiproxy_traffic_report` in your application or tool development.
+    > The table schema of `tiproxy_traffic_replay` might change in future versions. It is not recommended to directly read data from `tiproxy_traffic_replay` in your application or tool development.
+    > - Replay does not guarantee that the transaction execution order between connections is exactly the same as when it was captured, so it may report errors by mistake.
+    > - When replaying traffic again, the previous replay report will not be deleted automatically and needs to be deleted manually.
```

Comment:
```suggestion
    > - Replay does not guarantee that the transaction execution order between connections will exactly match the capture sequence. This might lead to incorrect error reports.
    > - TiProxy does not automatically delete the previous replay report when replaying traffic. You need to manually delete it.
```